.github/workflows/build.yml

```diff
@@ -3,19 +3,14 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        node-version: [18.x]
-        os: [ubuntu-latest]
+    runs-on: ubuntu-latest
```

Comment:
Removed matrix since only one option was used and it adds a bunch of clicks to get to the actual action you want to see. :)